src/components/books/book.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable jsx-quotes */
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook, getAllBooks } from '../../redux/books/books';
+
+const book = () => {
+  const post = useSelector((state) => state.bookReducer);
+  const dispatch = useDispatch();
+  const handleClick = (id, e) => {
+    e.preventDefault();
+    dispatch(removeBook(id));
+  };
+  useEffect(() => {
+    dispatch(getAllBooks());
+  }, []);
+
+  return (
+    <>
+      {post.map((book) => (
+        <div
+          key={book.item_id}
+          className="row"
+        >
+          <section className="grid-item">
+            <div>
+              <p><span className="category">{book.category}</span></p>
+              <p><span className="title">{book.title}</span></p>
+              <p><span className="author">{book.author}</span></p>
+            </div>
+            <ul className="list">
+              <li>
+                <button type="button">Comment</button>
+              </li>
+              <li>
+                <button
+                  className='cursor-pointer'
+                  onClick={(e) => handleClick(book.item_id, e)}
+                  type='button'
+                >
+                  Remove
+                </button>
+              </li>
+              <li>
+                <button type="button">Edit</button>
+              </li>
+            </ul>
+          </section>
```

Comment:
Zephin Design          |  Your Design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/69638013/168319370-faa8eb40-51fd-490d-b0a9-e02389321123.png)  |  ![Screenshot_183](https://user-images.githubusercontent.com/69638013/168319913-07dc8fa2-b18f-4255-97e4-1237cc553c27.png)

- [ ] Kindly add the border between the buttons and add border before the chapter section  
- This will make your design look exactly like the Figma design